content/blog/2019-01-06-babette.md

```diff
@@ -0,0 +1,205 @@
+---
+slug: babette
+title: Call BEAST2 from R
```

Comment:
Most readers won't know what BEAST2 is, so they might not read. I suggest revising title to be more informative. Something like "Call BEAST2 for Bayesian evolutionary analysis from R"?

Reply:
Thanks, I followed your suggestion :+1: